include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1705,7 +1705,8 @@ class MatrixFree : public Subscriptor
    * In order to apply different operators to different parts of the boundary,
    * this method can be used to query the boundary id of a given face in the
    * faces' own sorting by lanes in a VectorizedArray. Only valid for an index
-   * indicating a boundary face.
+   * indicating a boundary face. In all other cases,
+   * numbers::invalid_unsigned_int is returned.
```

Comment:
I don't think this is true for what gets returned? We return `exterior_face_no` which is typically the local face index (e.g. between 0 and 5 on 3D cube). Overall, I am not sure if this change makes sense as the value will not be usable.

Reply:
True. I have overlooked this.